mkdocs.yml

```diff
@@ -0,0 +1,49 @@
+site_name: Perseo Context-aware Complex Event Processing
+site_description: Perseo Documentation
+repo_url: https://github.com/Ficodes/perseo-fe
```

Comment:
Now should be https://github.com/telefonicaid/perseo-fe

Reply:
Ok. Fixed in 98273bc